ui/transactions.go

```diff
@@ -15,7 +30,128 @@ func (win *Window) TransactionsPage() {
 					}),
 				)
 			}),
+			layout.Flexed(.2, func() {
+				layout.Flex{Axis: layout.Horizontal}.Layout(win.gtx,
+					layout.Rigid(func() {
+						layout.Inset{Right: unit.Dp(10)}.Layout(win.gtx, func() {
+							win.combined.transactionSort.Layout(win.gtx, func() {
+							})
+						})
+					}),
+					layout.Rigid(func() {
+						win.combined.transactionStatus.Layout(win.gtx, func() {
+						})
+					}),
+				)
+			}),
+			layout.Flexed(.6, func() {
+				layout.UniformInset(unit.Dp(30)).Layout(win.gtx, func() {
+					txsList.Layout(win.gtx, len(win.combined.transactions), func(index int) {
+						layout.Inset{Bottom: unit.Dp(15)}.Layout(win.gtx, func() {
+							renderTxsRow(win, index)
+						})
+					})
+				})
+			}),
 		)
 	}
+
 	win.TabbedPage(bd)
 }
+
+func renderTxsRow(win *Window, index int) {
+	transaction := &win.combined.transactions[index]
+	if transaction.info == nil {
+		return
+	}
+
+	info := transaction.info
+	amountTxt1, amountTxt2 := parseAmount(info.Amount)
+
+	layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(win.gtx,
+		layout.Rigid(func() {
+			layout.Stack{}.Layout(win.gtx,
+				layout.Stacked(func() {
+					layout.Inset{Top: unit.Dp(10)}.Layout(win.gtx, func() {
+						background(win, false, info.Direction)
+					})
+				}),
+				layout.Stacked(func() {
+					layout.Inset{Left: unit.Dp(5)}.Layout(win.gtx, func() {
+						background(win, true, info.Direction)
+						layout.Inset{Left: unit.Dp(5), Top: unit.Dp(-2)}.Layout(win.gtx, func() {
+							transaction.iconDirection.Color = win.theme.Color.InvText
+							transaction.iconDirection.Layout(win.gtx, unit.Dp(16))
+						})
+					})
+				}),
+			)
```

Comment:
Just render the icon. We'll change to a more appropriate icon later

Reply:
Okay